api/src/test/resources/application.properties

```diff
@@ -54,3 +54,5 @@ spring.jpa.show-sql=true
 max.parallel.saga=10
 purge.records.saga.after.days=0
 scheduled.jobs.purge.old.saga.records.cron=-
+initialization.background.enabled=true
+url.api.institute=https://test
```

Comment:
Do we want this url link? Seems like it would never work.

Reply:
This is fine; the callout is mocked...this is just to satisfy sping injection